test/properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var jsc = require('jsverify');
```

Comment:
That's right. [FAQ][1]:

> ### Why do all the examples import the library as jsc instead of jsv?
>
> Does JSC originate with [JSCheck][2]?
>
> __A:__ Yes


[1]: https://github.com/jsverify/jsverify#why-do-all-the-examples-import-the-library-as-jsc-instead-of-jsv
[2]: http://www.jscheck.org/